src/plone/api/tests/test_group.py

```diff
@@ -480,17 +481,27 @@ def test_grant_roles_in_context(self):
         ROLES = set(['Editor', 'Contributor'])
         self.assertEqual(
             ROLES,
-            set(api.group.get_roles(
-                groupname='foo', obj=folder, inherit=False)),
+            set(
+                api.group.get_roles(
+                    groupname='foo',
+                    obj=folder,
+                    inherit=False,
+                ),
+            ),
         )
         self.assertEqual(
             ROLES,
-            set(api.group.get_roles(group=group, obj=folder, inherit=False)),
+            set(api.group.get_roles(group=group, obj=folder, inherit=False,)),
```

Comment:
Ok I could agree on that, but it seams also that the tailoring in plone.recipe.code-analysis does not enforce that, while the plugin itself in flake8 tries that.

I will remove that plugin for the moment from the plugins list in tox and remove stupid commas in the files.